apps/test/unit/signUpFlow/FinishStudentAccountTest.tsx

```diff
@@ -308,21 +308,59 @@ describe('FinishStudentAccount', () => {
     expect(screen.queryByText(locale.email_error_message())).toBe(null);
     expect(finishSignUpButton.getAttribute('aria-disabled')).toBe('true');
 
-    // Enter state
```

Comment:
Updating these comments since I realized they were wrong for this test (copied from the state input test)